cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/LowestAddressJoinDecider.scala

```diff
@@ -139,17 +139,25 @@ class LowestAddressJoinDecider(system: ActorSystem, settings: ClusterBootstrapSe
       def hostMatches(host: String, lowest: ResolvedTarget): Boolean =
         (host == lowest.host) || (Some(host) == lowest.address.map(_.getHostAddress))
 
-      // we check if a contact point is "us", by comparing host and port that we've bound to
-      def lowestContactPointIsSelfManagement(lowest: ResolvedTarget): Boolean = lowest.port match {
+      def matchesSelf(target: ResolvedTarget): Boolean = target.port match {
         case None =>
-          selfContactPoints.exists { case (host, _) => hostMatches(host, lowest) }
+          selfContactPoints.exists { case (host, _) => hostMatches(host, target) }
```

Comment:
Not this PR but very odd that selfContactPoints refers directly to Kubernetes API configuration. Users may have that but not be using kubernetes api for boostrap

Reply:
Yes, I agree but didn't find another way - (https://github.com/akka/akka-management/pull/238#discussion_r201627109)